app/lib/efile/nc/d400_calculator.rb

```diff
@@ -3,7 +3,19 @@ module Nc
     class D400Calculator < ::Efile::TaxCalculator
       attr_reader :lines
 
+      def initialize(year:, intake:, include_source: false)
+        super
```

Comment:
why do we need a `super` here? I guess I'm comparing these to the AZ322 calculators and wondering why this one takes different parameters...does it have to do with the fact this is the `main` calculator that will take in values from other calculators?

Reply:
how come it didn't call `super` before this PR?

Reply:
we will need to instantiate the new calculators in the initialize method. we did not have the d400 schedule s calculator before this PR